backend/ttrs/recommend.py

```diff
@@ -0,0 +1,42 @@
+import json
+import random
+
+
+def recommend(_options):
+    try:
+        options = json.loads(_options)
+
+        print(options)
+
+        recommends = {}
+        num_recommends = 3
+        for i in range(num_recommends):
+            lectures = build_timetable(options)
+            score = get_score(lectures, options)
+
+            recommends['tt {}'.format(i)] = {"lectures": lectures, "score": score}
```

Comment:
`'tt {}'` seems redundant because the frontend will expect to get time table list.
Therefore, recommends would be a list not a dictionary.

Reply:
Changed _recommends_ to list.